docs/src/pages/index.jsx

```diff
@@ -0,0 +1,217 @@
+                <p>
+                  Built with open source infrastructure and tools to give you
+                  complete flexibility over your deployment and fit your
+                  team&apos;s specific needs.
+                </p>
+              </li>
+            </ol>
+          </div>
+        </section>
+
+        <section className={styles.platformsSection}>
+          <div className={[styles.sectionContent, styles.platformsSectionContent].join(" ")}>
+            <div className={styles.platformsLeft}>
+              <div className={styles.platformsTextColumn}>
+                <h2
+                  className={[
+                    styles.sectionHeading,
+                    styles.platformsHeading,
+                  ].join(" ")}
+                >
+                  Deploy anywhere
+                </h2>
+                <p>
+                  Try Nebari on your local machine or deploy it on your cloud of
+                  choice. Nebari is designed to be flexible, extensible, and
+                  vendor-agnostic.
+                </p>
+                <p>
+                  Nebari can be seamlessly deployed to the major public cloud
+                  providers, including AWS, Azure, GCP, and Digital Ocean or to
+                  bare-metal HPC clusters.
+                </p>
+              </div>
+            </div>
+            <div className={styles.platformsRight}>
+              <ul className={styles.platformsList}>
+                <li className={styles.platformsItem}>
+                  <AwsSvg />
+                </li>
+                <li className={styles.platformsItem}>
+                  <AzureSvg />
+                </li>
+                <li className={styles.platformsItem}>
+                  <GcloudSvg />
+                </li>
+                <li className={styles.platformsItem}>
+                  <DigitalOceanSvg />
+                </li>
+              </ul>
+            </div>
+          </div>
+        </section>
+
+        <section className={styles.getStartedSection}>
+          <div className={styles.sectionContent}>
+            <h2 className={styles.sectionHeading}>Get started with Nebari</h2>
+            <p>
+              <a
+                href="/docs/category/get-started"
+                className={styles.getStartedLink2}
+              >
+                Learn how to deploy a Nebari instance&nbsp;&nbsp;
+                <RightArrowSvg />
+              </a>
+            </p>
+            <div>
+              <img src="/img/home/dashboard.png" alt="Nebari dashboard" />
+            </div>
+          </div>
+        </section>
+
+        <section className={styles.integrationsSection}>
+          <div className={styles.sectionContent}>
+            <h2 className={styles.sectionHeading}>Integrations</h2>
+            <p className={styles.integrationsSubheading}>
+              Nebari comes with out-of-the-box integrations to multiple tools in
+              the Data Science ecosystem.
```

Comment:
```suggestion
              the data science ecosystem.
```
This was my oversight